war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -198,8 +202,12 @@ public void renameOrganization(final String organizationId, final String newOrga
     *       id of the given organization to drop
     */
    public void dropOrganization(final String organizationId) {
-      dataStorage.dropDocument(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId));
       // TODO: check cannot be implemented without other facades changes
```

Comment:
Can we remove this TODO comment? Or is there still something that needs to be done here?